### ISSUE TITLE
BUG: WAVECAR deletion

### ISSUE THREAD

**Issue by JaGeo** [materialsproject/atomate2#769]
**Describe the bug**
In my current tests of the MPVaspLobsterMaker, the WAVECARs are not correctly deleted. It's a purely jobflow-based run. We would need to investigate this in detail.

@naik-aakash have you seen issues like this?

**Comment by naik-aakash** [comment]
Umm, I did not see this. But I can check it 😃

**Comment by JaGeo** [comment]
I ran the workflow with jobflow on supermuc. 

Could you also try running the workflow with jobflow remote? I get weird key errors when the lobster job is saved. It has nothing to do with size but with invalid keys.

**Comment by naik-aakash** [comment]
Is it realated to specifically to Bandoverlaps data key?

**Comment by JaGeo** [comment]
Mg3Sb2. The rest is not clear as the outputs are not saved. It would need further testing

**Comment by naik-aakash** [comment]
Thanks will look into it. 😄

**Comment by naik-aakash** [comment]
Hi @JaGeo ,Checked it with jobflow remote and it seems to originate from `band_overlaps `field. I will test with the recent version of our Schema (open PR), I think that should have fixed this issue already as we do not store it as dict anymore but as data object.

Seems earlier pydantic version did not have such checks implemented ,so never encountered this issue before.

**Comment by JaGeo** [comment]
I tried it with the recent version as well. It also failed for me

**Comment by naik-aakash** [comment]
Yeah , I just finished checking the same. Seems I need to update keys of bandoverlaps object in pymatgen to make it work with pydantic checks...
